README.md

```diff
@@ -35,7 +25,17 @@ Test files should use the following format:
 # include the body of an example test
```

Comment:
Either we need to come up with a generalized example or perhaps use the hello-world exercise test.

Reply:
Good catch, the standard practice is to use the Hello World